services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/permissions/HankeKayttajaController.kt

```diff
@@ -280,6 +284,11 @@ Responds with information about the activated user and the hanke associated with
                     responseCode = "404",
                     content = [Content(schema = Schema(implementation = HankeError::class))]
                 ),
+                ApiResponse(
+                    description = "Name not found in Profiili",
```

Comment:
The OpenAPI description can have only one response section per error code. Currently, this doesn't show up in the Swagger UI or the api-doc json, as it's being overwritten by the next response. ("Token doesn't have a user associated with it").

This description can be merged with that one.

Reply:
Good to know, thanks!